### PR TITLE
test: Convert remaining Cypress tests to Playwright

### PR DESCRIPTION
**Pull request by jcbcapps**
# SC-1397

## Proposed changes
- Converts the `auth.cy.js` and `saml.cy.js` Cypress tests into `portal-auth.spec.ts` and `saml.spec.ts` respectively
- Updates `myspace.spec.ts` to include add/remove News Section test
- Removes duplicate e2e tests
- Removes Cypress 

## Reviewer notes
Make sure that all tests are passing.

Note: this PR also contains the converted auth and saml tests mentioned in [this story](https://app.shortcut.com/orbit-truss/story/1396/convert-auth-and-saml-cypress-tests-to-playwright)

## Setup
Run the Playwright tests as you normally would:
- `yarn services:up -d`
- `yarn e2e:test`

### CODE REVIEW

File: e2e/playwright/feature/portal-auth.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        '/',
+        '/sites-and-applications',
+        '/about-us',
+        '/about-us/orbit-blog',
+        '/news',
+        '/news-announcements',
+        '/search',
+        '/settings',
+        '/ussf-documentation',
+      ]
```
Comment (gidjin) [inline e2e/playwright/feature/portal-auth.spec.ts:100]: question: Can this list be shared between this test and the first one?

File: e2e/playwright/portal-client/tests/myspace.spec.ts [hunk 2]
```diff
@@ -224,4 +224,23 @@ describe('MySpace', () => {
...
+    // Add News Section
+    await page.locator('text=Add section').click()
+    await page.locator('text=Add news section').click()
+    await expect(page.locator('text=Recent News')).toBeVisible()
+    await expect(page.locator('[data-testid="tag"] >> nth=0')).toBeVisible()
+
+    // Remove News Section
+    await page.locator('[aria-label="Section Settings"]').click()
+    await page.locator('text=Remove this section').click()
+    await page.locator('#removeSectionModal button:has-text("Delete")').click()
```
Comment (gidjin) [inline e2e/playwright/portal-client/tests/myspace.spec.ts:244]: Question: Is it possible to add an expectation that the section is no longer visible after this?

**Review by gidjin — APPROVED** [review]
LGTM, everything passed locally in 4.5 minutes! I left a couple questions but they are not blocking.

File: e2e/playwright/feature/portal-auth.spec.ts (hunk 1 above)
Comment (jcbcapps) [reply e2e/playwright/feature/portal-auth.spec.ts:100]: Good catch. Yeah I don't see why not. I'll make that change!

File: e2e/playwright/portal-client/tests/myspace.spec.ts (hunk 2 above)
Comment (jcbcapps) [reply e2e/playwright/portal-client/tests/myspace.spec.ts:244]: I noticed this yesterday while updating the modal tests for the refactor I've been working on. It'll be in an upcoming PR!

**Pull request merged**